components/Nav.js

```diff
@@ -18,6 +18,11 @@ const links = {
   Leaderboard: {
     href: '/leaderboard',
   },
+  Stories, Art, Toons: {
+    href: '/blog/category/138',
```

Comment:
Are you planning on populating this category? 

I would rather keep stories and art on separate categories and build in a filter to get those specific 2 categories for a proper implementation of what you're looking to achieve here.

Furthermore, category view is terribly broken beyond a single page of results, and needs to be fixed before we can start linking to individual categories.

Reply:
Yes, I intended to use this category as a pool for all posts of those three categories. the idea is to save space on the front page by using one link instead of three.